### DISCUSSION TITLE
active_sensing

### DISCUSSION THREAD

**Discussion by shawnthesheep** [mido/mido#405] [Q&A] [answered]
Hi, 
I am petty new to python and mido but it seems to me that
active_sensing messages are not recognized by mido.
I get pgm, c,c sysex ... messages as expected but no active_sensing (254 / 0xFE) bytes.
With pure data and the same setup I get active_sensing messages.
Is it me or a bug or whatever?
Could help somebody?
cheers

**Comment by rdoursenaud** [comment, marked as answer]
The default backend `mido` uses - `RtMidi` - ignores `active_sensing` messages by default.

If you really need them, you'd have to edit https://github.com/mido/mido/blob/main/mido/backends/rtmidi.py#L133 and change the last bool to `False`:
```python
self._rt.ignore_types(sysex=False, timing=False, active_sense=False)
```

**Reply by shawnthesheep** [reply]
Nice!
thanks
